src/guides/v2.4/comp-mgr/prereq/prereq-elasticsearch.md

```diff
@@ -0,0 +1,56 @@
+
+*  [Install and configure Elasticsearch]({{site.baseurl}}/guides/v2.3/config-guide/elasticsearch/es-overview.html)
+*  [Installing Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html)
+*  Configure Elasticsearch to work with [nginx]({{site.baseurl}}/guides/v2.3/config-guide/elasticsearch/es-config-nginx.html) or [Apache]({{site.baseurl}}/guides/v2.3/config-guide/elasticsearch/es-config-apache.html)
+*  [Configure Magento to use Elasticsearch]({{site.baseurl}}/guides/v2.3/config-guide/elasticsearch/configure-magento.html) and reindex
+
+Some third-party catalog search engines run on top of Magento's search engine. Contact your vendor to determine whether you need to update your extension.
+
+## Elasticsearch 2
+
+Magento does not support Elasticsearch 2. You must update to more recent version. Elasticsearch 7 is recommended.
+
+Refer to [Upgrading Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html) for full instructions on backing up your data, detecting potential migration issues, and testing upgrades before deploying to production. Depending on your current version of Elasticsearch, a full cluster restart may or may not be required.
+
+{:.bs-callout-info}
+Elasticsearch requires JDK 1.8 or higher. See [Install the Java Software Development Kit (JDK)]({{page.baseurl}}/install-gde/prereq/elasticsearch.html#prereq-java) to check which version of JDK is installed.
+
+[Configure Magento to use Elasticsearch]({{site.baseurl}}/guides/v2.3/config-guide/elasticsearch/configure-magento.html) describes the tasks you must perform after updating Elasticsearch 2.
+
+## Elasticsearch 5
+
+Elasticsearch 5 has reached its [End of Life](https://www.elastic.co/support/eol), and Magento support for Elasticsearch 5 has been deprecated.
+
+Although you can upgrade to Magento 2.4 without updating Elasticsearch 5, Magento recommends that you update to Elasticsearch 7.
+
+Refer to [Upgrading Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html) for full instructions on backing up your data, detecting potential migration issues, and testing upgrades before deploying to production. Depending on your current version of Elasticsearch, a full cluster restart may or may not be required.
+
+[Configure Magento to use Elasticsearch]({{site.baseurl}}/guides/v2.3/config-guide/elasticsearch/configure-magento.html) describes the tasks you must perform after updating Elasticsearch 5.
+
+## Elasticsearch 6 or 7
+
+If you are using Elasticsearch 6 or 7 as your catalog search engine, you are not required to perform any additional steps before upgrading to Magento 2.4.
+
+## Third-party extensions
+
+Magento recommends that you contact your search engine vendor to determine whether your extension is fully compatible with Magento 2.4. However, you are not required to perform any additional steps before upgrading to Magento 2.4.
```

Comment:
Can we say the following with confidence after asking them to contact their vendors about compatibility?

> However, you are not required to perform any additional steps before upgrading to Magento 2.4.0

Reply:
That makes sense. That last sentence still seems like a liability. I recommend removing it, but I don't insist on it.